README.md

```diff
@@ -110,17 +128,29 @@ ROLLBAR_ACCESS_TOKEN_KEY="The xcconfig file's key with which I store the Rollbar
       [optional: if empty, it will act as if you don't use rollbar; if completed, it will upload the dsym file to Rollbar server when deploying]
 ```
 
-For any other task you want to do, you can add them to the `Fastfile`s existing lanes or the custom actions.
-For example, if you want to have the script take care of distributing the build, you have to change `skip_waiting_for_build_processing` to `false` for pilot in the `Fastfile.private` and then add the distributing part to the script.
-For any further configuration, you will have to
+### Android
+```
+APP_NAME="Name of the app"
+
+FIREBASE_GROUPS_PATH="Groups that will be assigned to releases, configured in App Distribution
+
+PLAY_CONSOLE_KEY_FILE_PATH="Path of the key file used to deploy to Play Console"
+
+RUNNING_ON_CI="(true/false) If true, all interactive messages will be skipped"
+
+PACKAGE_NAME="Package name of the app, for example: com.wolox.app"
+
+FIREBASE_APP="ID of your project in Firebase, for example: 1:1234567890"
```

Comment:
Is missing here how firebase authenticates, with `FIREBASE_TOKEN` env var or with login on the CLI

Reply:
? we are not using firebase_token as an env var